### PR TITLE
Fix morgue deleting people out of existence

### LINKED ISSUE #34538: Dismantling a morgue with someone in it deletes that person
(WEB REPORT BY: apotheoticorder REMOTE: 206.221.180.138:7777)
# Revision
d6d7449ffd0b3294d101ab38007e2283fe9a862a

# Description

What in the fuck

### PR DESCRIPTION
**Pull request by Securitymain**
[bugfix]

## What this does

Fixes morgue dismantle action deleting people

closes #34538
## Why it's good

No more banishing your metagrudge to the shadowrealm

## Changelog

:cl:
 * bugfix: Morgues now drop their contents when dismantled

### CODE REVIEW

File: code/game/objects/structures/morgue.dm [hunk 1]
```diff
@@ -144,20 +144,20 @@
...
 			new /obj/structure/closet/body_bag(src.loc)
-			new /obj/item/stack/sheet/metal(src.loc,5)
+			new /obj/item/stack/sheet/metal(src.loc, 5)
 			qdel(src)
 	if(P.is_wrench(user))
 		P.playtoolsound(src, 50)
-		if(dir==4)
-			dir=8
+		if(dir == 4)
+			dir = 8
```
Comment (Securitymain) [inline code/game/objects/structures/morgue.dm:160]: are we allowed to change format things like this in prs?? i hope so cuz i hate this style

Comment (Inorien) [reply code/game/objects/structures/morgue.dm:160]: yeah, i find it being spread out like that is much easier to read
shame it kinda clutters the diff tho

**Comment by kacurasuo** [comment]
it would be better if it would just puke its contents on the ground instead of being undeconstructable, but I get that that's the previous behavior so nbd

**Comment by Securitymain** [comment]
> it would be better if it would just puke its contents on the ground instead of being undeconstructable, but I get that that's the previous behavior so nbd

grafik
its done

File: code/game/objects/structures/morgue.dm (hunk 1 above)
Comment (Kurfursten) [reply code/game/objects/structures/morgue.dm:160]: This kind of thing is fine

**Comment by Kurfursten** [comment]
Update CL

**Comment by Securitymain** [comment]
Since crematorium trays have the same problem I'm fixing that in a bit

**Pull request merged**